cms/djangoapps/contentstore/views/item.py

```diff
@@ -294,6 +294,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
+            is_library_view = view_name == 'container_child_preview'
```

Comment:
@cahrens Ah, perhaps the naming of the variable is a little confusing: The library root block itself uses `container_preview` as a view name. Child blocks that are added to a library use `container_child_preview`. That view name was [added](https://github.com/edx/edx-platform/pull/6459/files#diff-a4322d0c318325d7e9bf6cdb916827ceR217) as part of the main content library implementation (#6459), and doesn't seem to be used anywhere else.

I just checked, components inside of content experiments don't use `container_child_preview` (like child blocks on unit pages, they use `reorderable_container_child_preview`).